### PR TITLE
Release 0.1.

### PR DESCRIPTION
**Pull request by facundobatista**
Including several files and changes to make releases work.

### CODE REVIEW

File: HOWTO_RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Steps
+
+Release early, release often. Don't be lazy.
+
+To use this doc: just replace 0.1.0 with the major.minor.patch version of
+the release. The sequence of commands below should be good to copy and
+paste, but do please pay attention to details!
```
Comment (cmatsuoka) [inline HOWTO_RELEASE.md:7]: I believe this was meant to be "please do" instead of "do please"?

File: HOWTO_RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- cherry pick the needed commits from main:
+
+   git cherry-pick -m 1 COMMIT-HASH
+   ...
+
+- create release notes from the selected commits
+
+    git log
+
+- tag the release (using those release notes)
```
Comment (cmatsuoka) [inline HOWTO_RELEASE.md:55]: How do I tag using release notes?

**Review by cmatsuoka — APPROVED** [review]
Thanks, it's good to have a document with release information. I must add that to craft-parts as well.

General structure looks good, I just added a couple of notes to verify before merging.

File: HOWTO_RELEASE.md (hunk 2 above)
Comment (facundobatista) [reply HOWTO_RELEASE.md:55]: As indicated in the next command (it will open you an editor).

File: HOWTO_RELEASE.md (hunk 1 above)
Comment (facundobatista) [reply HOWTO_RELEASE.md:7]: Just removed the `do`. And also replaced the numbers for generic X.Y.Z (was a leftover).

**Pull request merged**